### DISCUSSION TITLE
Application Launcher

### DISCUSSION THREAD

**Discussion by tschissler** [Wendelpunkt/ToypadChallenge#25] [Ideas]
Hi, TyopadChallenge Community,

Ich würde das Toypad gerne nutzen um über verschiedene Tokens entsprechende Anwendungen auf dem PC starten zu können. 

* Die Felder können dabei verschiedene Monitore repräsentieren. Damit aknn ich also steuern, auf welchem Monitor die Anwendung erscheinen soll. 
* Werden mehrere Tokens auf einem Feld platziert, dann sollten die Anwendungen nebeneinander auf diesem Bildschirm erscheinen. 
* Wird der Token entfernt, so wird die Anwendung minimiert.

**Comment by tomwendel** [comment]
Je mehr ich drüber nachdenke, umso cooler finde ich die Idee
